package.json

```diff
@@ -1,42 +1,57 @@
 {
   "name": "tree-sitter-kotlin",
   "version": "0.3.7",
-  "description": "Tree-sitter grammar for Kotlin",
+  "description": "Kotlin grammar for tree-sitter",
+  "repository": "github:fwcd/tree-sitter-kotlin",
+  "author": "fwcd",
+  "license": "MIT",
   "main": "bindings/node",
-  "scripts": {
-    "test": "tree-sitter test",
-    "install": "node-gyp rebuild",
-    "generate": "tree-sitter generate",
-    "parse": "tree-sitter parse",
-    "build-wasm": "tree-sitter build-wasm",
-    "playground": "tree-sitter playground"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/fwcd/tree-sitter-kotlin.git"
-  },
+  "types": "bindings/node",
   "keywords": [
+    "incremental",
+    "parsing",
     "tree-sitter",
-    "kotlin",
-    "grammar"
+    "kotlin"
+  ],
+  "files": [
+    "grammar.js",
+    "binding.gyp",
+    "prebuilds/**",
+    "bindings/node/*",
+    "queries/*",
+    "src/**",
+    "*.wasm"
   ],
-  "author": "fwcd",
-  "license": "MIT",
-  "gypfile": true,
-  "bugs": {
-    "url": "https://github.com/fwcd/tree-sitter-kotlin/issues"
-  },
-  "homepage": "https://github.com/fwcd/tree-sitter-kotlin#readme",
   "dependencies": {
-    "nan": "^2.19.0"
+    "node-addon-api": "^8.0.0",
+    "node-gyp-build": "^4.8.1"
   },
   "devDependencies": {
-    "tree-sitter-cli": "^0.22.1"
+    "prebuildify": "^6.0.1",
+    "tree-sitter-cli": "^0.22.6"
```

Comment:
This updates tree-sitter to 0.22.6. Updating it is good, but we should make sure that the playground on the `gh-pages` branch stays in sync, I remember having some issues with that in the past. Could you check that the Wasm binary still works with the playground (potentially after upgrading `web-tree-sitter` too)?

Reply:
It seems to work. FWIW I do want to add a deployable version of the playground to the CLI in the future.